### DISCUSSION TITLE
Parallel processing

### DISCUSSION THREAD

**Discussion by sajutabraham** [neperfepx/FEPX#23] [Q&A] [answered]
While running fepx, it shows "Running on 1 cores". How to run the fepx in multiple cores? The program was set using the command "$ mpirun -np 4 fepx". Please excuse me as I am a beginner in this field but trying to explore. I request your help.

**Comment by mkasemer** [comment, marked as answer]
@sajutabraham,

If you are planning to submit to a small, one-node machine (i.e., a workstation, desktop, or laptop computer), you commonly use the `mpirun` command as you described.

When submitting on a multi-node computational cluster, you will often use a scheduler to submit your jobs (i.e., slurm or torque).

Either way, the basics of either option are covered in the FEPX documentation, which I suggest you consult. The documentation details necessary dependencies, as well as how to run in parallel on either type of system.

**Reply by sajutabraham** [reply]
Thank you. Let me read and understand.
